src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/HtmlTag.java

```diff
@@ -534,12 +534,10 @@ public String getText() {
     }
 
     public static enum AttrKind {
-        HTML4,
-        HTML5,
+        OK,
         INVALID,
         OBSOLETE,
-        USE_CSS,
-        ALL
+        UNSUPPORTED
     }
```

Comment:
Is there a useful distinction between INVALID / OBSOLETE / UNSUPPORTED ?

Reply:
Same distinction as mentioned for ElemKind